addons/account/models/account_journal_dashboard.py

```diff
@@ -72,6 +72,26 @@ def _get_json_activity_data(self):
     show_on_dashboard = fields.Boolean(string='Show journal on dashboard', help="Whether this journal should be displayed on the dashboard or not", default=True)
     color = fields.Integer("Color Index", default=0)
     entries_count = fields.Integer(compute='_compute_entries_count')
+    has_sequence_holes = fields.Boolean(compute='_compute_has_sequence_holes')
+
+    def _query_has_sequence_holes(self):
+        self.env.cr.execute("""
```

Comment:
You are making an sql query even if `self` is empty. You should return an empty list directly in that case

Reply:
But since it's for a computed field, it is never called on an empty recordset, and it doesn't fail if for some reason it is called on one either, only with a very very short query